cluster/addons/kube-network-policies/kube-network-policies.yaml

```diff
@@ -45,11 +45,6 @@ spec:
         - mountPath: /lib/modules
           name: lib-modules
           readOnly: true
-        env:
-        - name: MY_NODE_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: spec.nodeName
```

Comment:
under the assumption that we shouldn't need to override it because the default will be correct?

Reply:
yes, I checked and IIUIC this is how we do in kube-proxy in CI too, I should have noticed before that kube-rpoxy does the same by default